vendor/golang.org/x/sync/errgroup/errgroup.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package errgroup provides synchronization, error propagation, and Context
+// cancelation for groups of goroutines working on subtasks of a common task.
+package errgroup
```

Comment:
Nice.  Note that #412 also adds this package, so one of us may have to deal with a merge conflict here.  Or maybe it will just work.